ocs_ci/deployment/deployment.py

```diff
@@ -164,6 +164,27 @@ def post_ocp_deploy(self):
             )
         # end of workaround
         self.add_stage_cert()
+        # In order to be able to deploy from stage we need to change
+        # image registry config to Managed state.
+        # More described in BZs:
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1806593
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1807471#c3
+        # We need to change to managed state as described here:
+        # https://github.com/red-hat-storage/ocs-ci/issues/1436
+        # So this is not suppose to be deleted as WA case we really need to do
+        # this operation for OCS deployment as was originally done here:
+        # https://github.com/red-hat-storage/ocs-ci/pull/1437
+        # Currently it has to be moved here to enable CA certificate to be
+        # properly propagated for the stage deployment as mentioned in BZ.
+        if(config.ENV_DATA['platform'] not in constants.CLOUD_PLATFORMS):
+            run_cmd(
+                f'oc patch {constants.IMAGE_REGISTRY_CONFIG} --type merge -p '
+                f'\'{{"spec":{{"storage": {{"emptyDir":{{}}}}}}}}\''
+            )
+            run_cmd(
+                f'oc patch {constants.IMAGE_REGISTRY_CONFIG} --type merge -p '
+                f'\'{{"spec":{{"managementState": "Managed"}}}}\''
+            )
```

Comment:
This is quite a lot of information for individual comment lines. Would it be better to create a function that we call here instead and have this information in the docstring? This has the additional benefit of propagating this information to our documentation.

Reply:
Thanks for suggestion. I incorporated this in next commit.